syntax/apiblueprint.vim

```diff
@@ -20,6 +22,8 @@ syntax region apibActionRelation start=/^[-+*] Relation: .*$/ end=/$/ oneline co
 syntax match apibHeadersSectionKey /^[^:]*\ze:/ contained
 syntax match apibHeadersSectionValue /:.*/ contained
 
+syntax region apibJSON start=/^\s*{/ end=// contained containedin=markdownCodeBlock contains=@json
```

Comment:
Thanks for the pull request, I though I already commented on this but must've forgot.

Would you be able to update this to also support arrays?

I wonder if we can check for `end` too, to make sure the code-block ends the same way it started (i.e `{}`).